### DISCUSSION TITLE
Openvino in Intel GPU

### DISCUSSION THREAD

**Discussion by murraharri** [ggml-org/whisper.cpp#2650] [Q&A]
Iam very new to python and just tryhing transcription for my education purpose, i am trying to use openvino but no succcessful in it. I am doing as per the steps given for openvino usage. Below is one of the errors. Can someone kindly help how to getrid of this error?
---------------------------Error----------------------------

(openvino_conv_env) E:\whisper.cpp\models>python convert-whisper-to-openvino.py --model base.en
100%|███████████████████████████████████████| 139M/139M [00:04<00:00, 33.9MiB/s]
E:\whisper.cpp\models\openvino_conv_env\lib\site-packages\whisper\__init__.py:150: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  checkpoint = torch.load(fp, map_location=device)
E:\whisper.cpp\models\openvino_conv_env\lib\site-packages\whisper\model.py:197: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert x.shape[1:] == self.positional_embedding.shape, "incorrect audio shape"
Traceback (most recent call last):
  File "E:\whisper.cpp\models\openvino_conv_env\lib\site-packages\torch\onnx\_internal\onnx_proto_utils.py", line 215, in _add_onnxscript_fn
    import onnx
  File "E:\whisper.cpp\models\openvino_conv_env\lib\site-packages\onnx\__init__.py", line 77, in <module>
    from onnx.onnx_cpp2py_export import ONNX_ML
ImportError: DLL load failed while importing onnx_cpp2py_export: A dynamic link library (DLL) initialization routine failed.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "E:\whisper.cpp\models\convert-whisper-to-openvino.py", line 60, in <module>
    convert_encoder(hparams, encoder, args.model)
  File "E:\whisper.cpp\models\convert-whisper-to-openvino.py", line 24, in convert_encoder
    torch.onnx.export(
  File "E:\whisper.cpp\models\openvino_conv_env\lib\site-packages\torch\onnx\__init__.py", line 375, in export
    export(
  File "E:\whisper.cpp\models\openvino_conv_env\lib\site-packages\torch\onnx\utils.py", line 502, in export
    _export(
  File "E:\whisper.cpp\models\openvino_conv_env\lib\site-packages\torch\onnx\utils.py", line 1640, in _export
    proto = onnx_proto_utils._add_onnxscript_fn(
  File "E:\whisper.cpp\models\openvino_conv_env\lib\site-packages\torch\onnx\_internal\onnx_proto_utils.py", line 217, in _add_onnxscript_fn
    raise errors.OnnxExporterError("Module onnx is not installed!") from e
torch.onnx.OnnxExporterError: Module onnx is not installed!
---------------------------Error----------------------------

**Comment by pharmacologic** [comment]
> raise errors.OnnxExporterError("Module onnx is not installed!") from e
torch.onnx.OnnxExporterError: Module onnx is not installed!

you may need to do something like `pip install onnx` from within your `openvino_conv_env` python environment
